Makefile

```
@@ -86,12 +108,6 @@ upgrade:
 	@# A copy of the binary will also be placed under: ./bin/${PNAME}/${CTLNAME}
 	@PNAME=${UPGRADE} CTLNAME=${UPGRADE} CGO_ENABLED=0 sh -c "'$(PWD)/build/build.sh'"
 
-ifeq (${IMAGE_TAG}, )
-  IMAGE_TAG = ci
-  export IMAGE_TAG
-endif
-
-
 # build upgrade image
 .PHONY: upgrade-image.amd64
 upgrade-image.amd64: upgrade
```

```suggestion
.PHONY: upgrade-image
upgrade-image: upgrade
```
We dont need to specify arch anymore.